options.html

```diff
@@ -57,6 +61,10 @@
           <input id="speedStep" type="text" value=""/>
       </div>
       <div class="row">
+          <label for="fastSpeed">Preferred Fast Playback Speed (x)</label>
```

Comment:
Ditto, "Preferred Speed". 

(and I just realized we flip casing between shortcuts and settings.. I'll fix this later)